nyaa/templates/_formhelpers.html

```diff
@@ -139,3 +139,54 @@
 		{% endif %}
 	</div>
 {% endmacro %}
+
+{% macro render_category_select(name) %}
```

Comment:
this is nice, but my main concern is that in layout.html we have dupe code for setting categories. would it be easy to factor it out so they use the same macro?